sirepo/package_data/static/js/raydata.js

```diff
@@ -141,8 +141,13 @@ SIREPO.app.factory('raydataService', function(appState, panelState, requestSende
     };
 
     self.updateScanInfoTableColsInCache = function(cols) {
-        simulationDataCache.scanInfoTableCols = cols;
-        return cols;
+        if (cols.length) {
```

Comment:
Allowing simulationDataCache.scanInfoTableCols to be empty/undefined was causing the analysisStatus panel to be empty when the search query returned no scans while some scans were still checked. You can see what I mean if you check out 4b1993d72ae35b2ac185fae6990eee18b822c621, search for scans, select some, then make a search query that has no results (for example, start and end time the same), then visit the analysis tab.